Tests/ApiTests/ApiTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ApiTests\ApiTest;
+
+use App\Database\DatabaseConnection;
+use App\Database\QueryBuilder\MySqlQueryBuilder;
+use App\Repositories\WordRepository;
+use App\Services\FileService;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+
+class ApiTest extends TestCase
```

Comment:
ApiTest is too abstract I would expect here some abstract class that WordTest would extend. And most of this logic would belong to WordTest case (or maybe WordApiTest).

Also I would highly recommend to rewrite test in that way - by creating abstract `ApiTestCase` and move logic that is visibly common (like database handling, or api calls) to that abstract class.